libraries/CurieBLE/src/internal/BLEDeviceManager.cpp

```diff
@@ -576,10 +574,32 @@ BLEDevice BLEDeviceManager::peripheral()
     return temp;
 }
 
-bool BLEDeviceManager::startScanning()
+void BLEDeviceManager::_clearAdvertiseBuffer()
```

Comment:
The BLEDeviceManager.* doesn't need change. Those changes are for scan response. Jira876.